src/common.js

```diff
@@ -209,7 +209,7 @@ function getRowValue(row, position) {
   return null;
 }
 
-function getRowWithValueAtPosition(workSheet, value, namePosition = 1) {
+function getRowWithValueAtPosition(workSheet, value, namePosition = 1, rowType = undefined) {
```

Comment:
```suggestion
function getRowWithValueAtPosition(workSheet, value, namePosition = 1, rowType) {
```
I think this will be undefined if not provided

Reply:
It seems that the is that it continue searching after it found the "RowWithValueAtPosition". Do you think it will work with 
```
if (index === -1) {

}
```
in "workSheet.eachRow"?

Reply:
Interesting! I'm checking it out . 
Thanks for pointing that